app_dart/lib/src/service/github_service.dart

```diff
@@ -99,4 +99,25 @@ class GithubService {
         ..head = (PullRequestHead()..sha = commit['head']['sha'] as String);
     }).toList();
   }
+
+  // Returns changed files of [slug] and [prNumber].
+  // https://developer.github.com/v3/pulls/#list-pull-requests-files
+  Future<List<String>> listFiles(RepositorySlug slug, int prNumber) async {
```

Comment:
Isn't this list of files already available in the PR event?

Reply:
Doesn't seem so. Based on `CheckSuiteEvent`, we can track down to `CheckSuite`, then `PullRequest`, where `changedFilesCount` is available, but there is no `changedFiles` though.